### PR TITLE
Done some changes in css file (#3672)

### LINKED ISSUE #3672: Logo on the homepage is glitchy on first load
## Expected Behavior
When I load https://dev-knative.netlify.app/ I should see a the banner without any glitches

## Actual Behavior
When you first load the page, you see the knative logo flash against a white background before being put in the banner where it is supposed to render

## Steps to Reproduce the Problem

1. Go to https://dev-knative.netlify.app/
1. Switch between any tab and the homepage quickly
1. See the bug 

## Additional Info
ezgif-6-060cd997d6c4

### PR DESCRIPTION
**Pull request by soham283**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-Done some css related changes
-Fixes #3672 
-

### CODE REVIEW

**Comment by omerbensaadon** [comment]
Unfortunately, this doesn't fix the issue. See the GIF Below: 
ezgif-6-060cd997d6c4

**Comment by soham283** [comment]
I will check with another solution

**Comment by soham283** [comment]
@googlebot I fixed it.

**Comment by soham283** [comment]
@googlebot I fixed it.

**Comment by soham283** [comment]
@googlebot I fixed it.

**Comment by soham283** [comment]
@googlebot I fixed it.

File: overrides/assets/stylesheets/home.css [hunk 1]
```diff
@@ -47,7 +47,10 @@
   background-size: 300% 100%;
   opacity: 0.4;
   z-index: -1;
-  padding: 20px;
+  padding : 20px;
+}
+.animated-gradient img{
+display: block !important;
```
Comment (julz) [inline overrides/assets/stylesheets/home.css:53]: nit: let's fix the formatting up here so it's indented and spaced like the rest of the file:

```suggestion
  padding: 20px;
}
.animated-gradient img {
  display: block !important;
```

File: overrides/home.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <div class="md-container">
   <section class="home-container">
     <div class="animated-gradient">
-      <img src="images/logo/rgb/knative-logo-rgb.png" />
+        <img src="images/logo/rgb/knative-logo-rgb.png" style="display: none;"/>
```
Comment (julz) [inline overrides/home.html:7]: let's keep the indentation like it was before here

**Review by julz — COMMENTED** [review]
looks good to me! Just a couple formatting nits

File: overrides/home.html (hunk 2 above)
Comment (soham283) [reply overrides/home.html]: Done with the changes

File: overrides/assets/stylesheets/home.css [hunk 3]
```diff
@@ -49,6 +49,9 @@
   z-index: -1;
   padding: 20px;
 }
+.animated-gradient img{
```
Comment (julz) [inline overrides/assets/stylesheets/home.css:52]:
```suggestion
.animated-gradient img {

```

File: overrides/assets/stylesheets/home.css [hunk 4]
```diff
@@ -49,6 +49,10 @@
   z-index: -1;
   padding: 20px;
 }
+.animated-gradient img {
+
```
Comment (julz) [inline overrides/assets/stylesheets/home.css:53]: looks like an extra newline crept in here?

Comment (julz) [reply overrides/assets/stylesheets/home.css:53]: oh no! that was my fault in the github suggestion - sorry 😬. Still can we drop it, and then this looks good

**Review by julz — APPROVED** [review]
/lgtm
/approve

**Comment by julz** [comment]
Thanks @soham283!

**Comment by soham283** [comment]
Is the PR has been approved or not, the bot is showing Not approved

**Comment by soham283** [comment]
@googlebot I fixed it.

**Comment by soham283** [comment]
@googlebot I fixed it.

**Comment by julz** [comment]
@googlebot I fixed it.

**Pull request closed without merging**